### ISSUE TITLE
Downstream cancer classifier checkpoint

### ISSUE THREAD

**Issue by egistific** [batmanlab/Mammo-CLIP#31]
Hi,
May I check if the downstream cancer classifier checkpoint Downstream_evalualtion_b5_fold0/classification/Models/Classifier/fine_tune/cancer/upmc_breast_clip_det_b5_period_n_ft_seed_10_fold0_best_aucroc_ver084 (https://huggingface.co/shawn24/Mammo-CLIP/blob/main/Downstream-checkpoints/Downstream_evalualtion_b5_fold0.zip) is obtained by pretraining the Mammo-CLIP model on the UPMC dataset followed by finetuning on the RSNA dataset?
Thank you.

**Comment by shantanu-ai** [comment]
I did not get ur query right. So, the above checkpoint was obtained by pretraining the Mammo-CLIP model on the UPMC dataset followed by various downstream tasks. The downstream tasks include both linear probe and finetuning of detection and classification tasks. The detection tasks include detecting mass, calcification from VinDr dataset. The classification tasks include classifying mass, calcification and density from VinDr dataset and cancer from RSNA dataset. If you unzip the directory, u will see the corresponding folder structure with the task name (detection/classification) followed by the label name (mass, calcification, density, cancer).
